networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/config/GossipScoringConfigurator.java

```diff
@@ -135,8 +131,8 @@ public void configureDynamicTopics(final GossipTopicsScoringConfig.Builder build
 
     private void configureVoluntaryExitTopic(final GossipTopicsScoringConfig.Builder builder) {
       final String topic =
-          TopicNames.getTopic(
-              forkDigest, VoluntaryExitGossipManager.TOPIC_NAME, eth2Context.getGossipEncoding());
+          GossipTopics.getTopic(
+              forkDigest, GossipTopicNames.VOLUNTARY_EXIT, eth2Context.getGossipEncoding());
```

Comment:
Unrelated to this PR but do we need to configure scoring stuff for the new topics in Altair?

Reply:
Yes - I don't think we've integrated any of the new topics here